docs/advanced-topics/group-management.md

```diff
@@ -0,0 +1,90 @@
+
+In general, we support 5 methods for group management for all SDK and REST API for Azure SignalR Service:
+
+|  | [Server SDK](https://www.nuget.org/packages/Microsoft.Azure.SignalR/) | [Management SDK](https://www.nuget.org/packages/Microsoft.Azure.SignalR.Management) | [Function Binding](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.SignalRService) | [REST API](https://github.com/Azure/azure-signalr/blob/dev/docs/rest-api.md) |
+| --- | --- | --- | --- | --- |
+| Add a connection ID to a group | :heavy_check_mark: |  :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Remove a connection ID from a group | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Add a user ID from a group | `N/A` | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Remove a user ID from a group | `N/A` | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Remove a user ID from all groups | `N/A` |  :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Check user ID existence in a group | `N/A` |  `N/A` | `N/A` | :heavy_check_mark: |
+
+## Group management for connection ID
+
+To add or remove connection IDs from a group, you call the add or remove methods, and pass in the connection ID and group's name as parameters. Group membership **IS NOT** preserved when a connection ends. The connection needs to rejoin the group when it's re-established.
+
+At the momment, Azure SignalR Service doesn't provide any methods to check connection membership in a group. Therefore, it is app server's responsibility to manage the connection membership.
+
+### Adding and removing connection IDs
+
+#### Usage
+
+* [Server SDK Usage](https://docs.microsoft.com/en-us/aspnet/core/signalr/groups?view=aspnetcore-3.0#groups-in-signalr)
+* [Management SDK Usage](../management-sdk-guide.md#iservicehubcontext)
+* [Function Binding Usage](https://github.com/Azure/azure-functions-signalrservice-extension#using-the-signalr-output-binding)
+* [REST API Usage](../rest-api.md)
+
+## Group management for user ID
+
+To add or remove user IDs from a group, you call the add or remove methods, and pass in the user ID and group's name as parameters. Unlike connection ID, Group membership **IS** preserved when a user disconnects.
+
+Note that user ID can be viewed as a **tag** to one or more connections. If you add a new tag to a group, that means you notify the group that all the group messages should be sent to the connetion with the tag. It is your responsibility to update the tags (user IDs) in a group.
+
+We provides 2 options to clear tags in groups:
+
+1. [Add a user ID to a group with `TTL`](#adding-and-removing-user-ids).
+2. [Excipitly remove a user ID from all groups](#remove-user-from-all-groups).
+
+When you can decide the TTL of a user, you can use the [Add a user ID to a group with `TTL`](#adding-and-removing-user-ids) option. 
```

Comment:
Explain TTL a little bit. I think it might be confusing that TTL only impacts new connections but not existing connections.